Models/data_cleansing.py

```diff
@@ -87,14 +87,22 @@ def _create_dummies_for_categorical_features(design_matrix):
     return design_matrix
 
 
-def clean_data(design_matrix):
+def clean_data(design_matrix, remove_outliers=False):
     """Cleaning raw data before model processing."""
     design_matrix = _impute_missing_values(design_matrix)
     # design_matrix = _normalize_continuous_features(design_matrix)
     design_matrix = _create_dummies_for_categorical_features(design_matrix)
+    if remove_outliers:
+        design_matrix = outlier_treatment(design_matrix)
     return design_matrix
 
 
+def outlier_treatment(design_matrix, m=2.75):
```

Comment:
I'm not merging your updated PR right now. Let us first check if we can overcome this issue in any way.